packages/compiler/lib/projected-names.cadl

```diff
@@ -0,0 +1,100 @@
+// Set of projections consuming the @projectedName decorator
+
+#suppress "projections-are-experimental"
+projection op#wire {
```

Comment:
Hmm, are we thinking both serialization and language-specific projections will go here.  If so #wire is probably not the right name

Reply:
yeah it should be for every target. So really not sure what a good name would be.

```
wire: json, xml
lang: csharp, js, python
kind: server, client
```